.opencode/agent/protondrive-conventions.md

```
@@ -10,6 +10,42 @@ mode: primary
 - Repo: `donniedice/protondrive-linux`
 - Default branch: `main`
 - **Never push directly to `main`** — always use Issue → Branch → PR → Merge
+- No intermediate branches (no `alpha`) — feature branches merge directly to `main`
+
+## Workflow Execution Protocol
+
+### Step 1: Create Issue
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)